invalidPropsals.ts

```diff
@@ -0,0 +1,122 @@
+import snapshot from '@snapshot-labs/snapshot.js';
+const dotenv = require('dotenv');
+dotenv.config();
+global['fetch'] = require('node-fetch');
+
+import db from './server/helpers/mysql';
+import { jsonParse } from './server/helpers/utils';
+import { spaces } from './server/helpers/spaces';
+import { archiveProposal, loadSpaces } from './server/helpers/adapters/mysql';
+
+const tableName = 'messages';
+const backuptableName = 'messages_backup';
+
+
+const init = async () => {
+    loadSpaces().then(async ensSpaces => {
+        console.log('ENS spaces', Object.keys(ensSpaces).length);
+
+        await backup();
+        await removeInvalidProposals();
+
+        console.log("\n\n\nProcess Completed!!\n\n\n")
+        process.exit();
+    });
+}
+
+const removeInvalidProposals = async () => {
+    try {
+        console.log(`\n\n\nStarted Removing Invalid Proposals from table ${tableName} \n\n\n`);
+        const query = `SELECT * FROM ${tableName} WHERE type = 'proposal'`
+        const data = await db.queryAsync(query, [])
+        console.log(`Total ${data.length} Proposals found`);
+
+        for(const proposal of data) {
+            proposal.payload = jsonParse(proposal.payload)
+            proposal.metadata = jsonParse(proposal.metadata)

```

Comment:
Some proposal dont have this field "metadata", it was introduced i think on msg version 0.1.3